src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/BlockReflectionTypeMapNode.cs

```diff
@@ -0,0 +1,94 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+
+using ILCompiler.Metadata;
+
+using Internal.TypeSystem;
+using Internal.NativeFormat;
+using Internal.Text;
+
+namespace ILCompiler.DependencyAnalysis
+{
+    /// <summary>
+    /// Represents a map between EETypes and metadata records within the <see cref="MetadataNode"/>.
```

Comment:
Seems like a copypaste of a different comment. Can you please update with what this is?